### PR TITLE
JDF-351 - Update existing Shrinkwrap examples to use 2.0.0-beta5 (BOM 1.0.7.CR3) and add a new shrinkwrap-resolver quickstart

### PR DESCRIPTION
**Pull request by rafabene**
Since Pete said that it is quit urgent I'm sending this PR with the existing quickstarts updated in one commit and the new quickstart in another commit.

### CODE REVIEW

**Comment by rafabene** [comment]
Update of kitchensink-deltaspike was missing on JDF-346. This PR includes kitchensink-deltaspike update to 0.4 too

**Comment by rafabene** [comment]
Added #522 modification to this quickstart

File: shrinkwrap-resolver/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+shrinkwrap-resolver: Demonstrate usage of Shrinkwrap resolver
+==============================================================
+Author: Rafael Benevides
+Level: Beginner
```
Comment (pmuir) [inline shrinkwrap-resolver/README.md:4]: I think it should be intermediate.

File: shrinkwrap-resolver/src/main/resources/META-INF/persistence.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pmuir) [inline shrinkwrap-resolver/src/main/resources/META-INF/persistence.xml:1]: Just wondering if you want to put these files in src/test as I guess you only use them intests.

**Comment by pmuir** [comment]
Looks good and ready for merge apart from the above.

File: shrinkwrap-resolver/src/main/resources/META-INF/persistence.xml (hunk 2 above)
Comment (rafabene) [inline shrinkwrap-resolver/src/main/resources/META-INF/persistence.xml:1]: Yes! You're right. This came from archetype and I was not sure if we should kept it or not.

**Comment by rafabene** [comment]
Merged

**Comment by kpiwko** [comment]
Late review comment: It looks good to me! :+1:

**Comment by rafabene** [comment]
:)

**Pull request closed without merging**